prov/efa/src/rdm/efa_rdm_peer.h

```diff
@@ -92,6 +92,12 @@ struct efa_rdm_peer {
 	int64_t num_read_msg_in_flight;
 };
 
+/**
+ * @brief check for peer's RDMA_READ support, assuming HANDSHAKE has already occurred
+ *
+ * @param peer[in]	A peer which we have already received a HANDSHAKE from
```

Comment:
Does your IDE highlight this syntax? Mine only works with
```
@param[in] peer
```

Reply:
Fixed.  Thank you.